### PR TITLE
Issue #3263120 by yaroslav-shandra: Hide extra fields after enabling Social Profile Manager Notes

### PR DESCRIPTION
**Pull request by YaroslavShandra88**
## Problem
After enabling Social Profile Manager Notes module at the edit profile page, not hidden field (field_manager_notes) from users for enable/disable possibility left a comment by content manager at profile page.

## Solution
Hide field(field_manager_notes) from form display.

## Issue tracker
https://www.drupal.org/project/social/issues/3263120
https://getopensocial.atlassian.net/browse/YANG-7016

## How to test
1. Enable Social Profile Manager Notes submodule.
2. Open page for editing profile.

## Screenshots
Screenshot from 2022-02-08 14-42-25

## Release notes
Hide field_manager_notes field after enabling Social Profile Manager Notes at edit profile page.

### CODE REVIEW

**Comment by open-social-tugboat** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr2753-b9f4k9s1yidqkcfzoaw5mkrqsyjawonn.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/62026b98046a4b8d29b17fc7

File: modules/social_features/social_profile/modules/social_profile_manager_notes/social_profile_manager_notes.module [hunk 1]
```diff
@@ -84,3 +87,34 @@ function social_profile_manager_notes_theme($existing, $type, $theme, $path) {
   ];
   return $items;
 }
+
+/**
+ * Implements hook_entity_view_display_alter().
+ */
+function social_profile_manager_notes_entity_view_display_alter(EntityViewDisplayInterface $display, array $context): void {
+  if ( $context['entity_type'] === 'profile'
```
Comment (ribel) [inline modules/social_features/social_profile/modules/social_profile_manager_notes/social_profile_manager_notes.module:95]: @YaroslavShandra88 Please fix the Coding Standards.

**Review by ribel — CHANGES_REQUESTED** [review]

File: modules/social_features/social_profile/modules/social_profile_manager_notes/social_profile_manager_notes.module (hunk 1 above)
Comment (YaroslavShandra88) [reply modules/social_features/social_profile/modules/social_profile_manager_notes/social_profile_manager_notes.module]: > @YaroslavShandra88 Please fix the Coding Standards.

Done.

**Comment by open-social-tugboat** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr2753-b9f4k9s1yidqkcfzoaw5mkrqsyjawonn.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/62026b98046a4b8d29b17fc7

**Comment by ribel** [comment]
@YaroslavShandra88 Please remove not needed rule from `phpstan.neon` as well.

**Comment by open-social-tugboat** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr2753-b9f4k9s1yidqkcfzoaw5mkrqsyjawonn.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/62026b98046a4b8d29b17fc7

**Comment by YaroslavShandra88** [comment]
> @YaroslavShandra88 Please remove not needed rule from `phpstan.neon` as well.

Done

**Review by ribel — APPROVED** [review]
@YaroslavShandra88 All looks good to me now!
@Ressinel could you please check as well?

**Comment by ribel** [comment]
@Ressinel could you please merge and cherry-pick this one?

**Comment by Ressinel** [comment]
@YaroslavShandra88 Could you rebase your PR from the **main** branch and leave only informative commits?

**Comment by YaroslavShandra88** [comment]
@Ressinel Done.

**Comment by open-social-tugboat** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr2753-b9f4k9s1yidqkcfzoaw5mkrqsyjawonn.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/62026b98046a4b8d29b17fc7

**Comment by Ressinel** [comment]
🍒 picked to 11.1.x and 11.0.x

**Pull request merged**